docs/root/configuration/http_conn_man/stats.rst

```diff
@@ -86,3 +86,30 @@ following statistics:
    downstream_rq_3xx, Counter, Total 3xx responses
    downstream_rq_4xx, Counter, Total 4xx responses
    downstream_rq_5xx, Counter, Total 5xx responses
+
+.. _config_http_conn_man_stats_per_codec:
+
+Per codec statistics
+-----------------------
+
+Each codec has the option of adding per-codec statistics. Currently only http2 has codec stats.
+
+Http2 codec statistics
+~~~~~~~~~~~~~~~~~~~~~~
+
+All http2 statistics are rooted at *http2.*
+
+
+.. csv-table::
+   :header: Name, Type, Description
+   :widths: 1, 1, 2
+
+   rx_reset, Counter, Total number of reset streams frames received by Envoy.
+   tx_reset, Counter, Total number of reset stream frames transmitted by Envoy.
+   header_overflow, Counter, Total number of headers going over the 64512 byte limit.
+   trailers, Counter, Total number of trailers seen on requests coming from downstream.
```

Comment:
Why is it useful to count trailers?

Reply:
goodness only knows, but it's incremented on the request path and not the response path, according to my asserts + testing.